portfolio/src/main/webapp/script.js

```diff
@@ -136,6 +144,26 @@ function getLoginStatus() {
   return fetch(url).then((response) => response.json());
 }
 
+function getDateFromTimestamp(timestamp) {
+  const date = new Date(timestamp);
+
+  const day = formatTwoDigits(date.getDay());
+  const month = formatTwoDigits(date.getMonth());
+  const year = date.getFullYear();
+  const dateStr = `${day}.${month}.${year}`;
```

Comment:
can we use something more international? RFC3339 is a good start, I really prefer greater timeunits to come first (e.g. YYYY-MM-DD)

Reply:
I made use of the toLocaleString method which takes care of that. I also specified the British format, this way the time is in 24-hour format.